introduction.md

```diff
@@ -1,2 +1,6 @@
 I'm tk. w00t!
 I'm enjoying the javascript and ruby paths.
+* JavaScript
+* Ruby
+* AngularJS
+* Ember
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?